wrappers/nodejs/index.js

```diff
@@ -3982,6 +4077,28 @@ const timestamp_domain = {
      * @type {Integer}
      */
     TIMESTAMP_DOMAIN_COUNT: RS2.RS2_TIMESTAMP_DOMAIN_COUNT,
+    /**
+     * Get the string representation out of the integer timestamp_domain type
+     * @param {Integer} domainVal the timestamp_domain type
+     * @return {String}
+     */
+    timestampDomainToString: function(domainVal) {
+      if (arguments.length !== 1) {
+        throw new TypeError('timestamp_domain.timestampDomainToString() expects 1 argument');
+      } else {
+        let i = checkStringNumber(arguments[0],
+            this.TIMESTAMP_DOMAIN_HARDWARE_CLOCK, this.TIMESTAMP_DOMAIN_COUNT,
+            timestampDomain2Int,
+            'timestamp_domain.timestampDomainToString() expects a number or string as the 1st argument', // eslint-disable-line
+            'timestamp_domain.timestampDomainToString() expects a valid value as the 1st argument');
+        switch (i) {
+          case this.TIMESTAMP_DOMAIN_HARDWARE_CLOCK:
+            return this.timestamp_domain_hardware_clock;
+          case this.TIMESTAMP_DOMAIN_SYSTEM_TIME:
+            return this.timestamp_domain_system_time;
```

Comment:
no default?

Reply:
The default is not added as the above checkStringNumber function has performed value range check and will throw exception, so I ignored that check. Anyway, it's always better to add it, in case our code changes, I will add it then.